hack/generate_new_pool_admins.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" -ne 2 ]]; then
+    echo "Illegal number of parameters"
+    exit 1
+fi
+
+
+TEAM=$1
+OWNERS=$2
+IFS=',' read -r -a OWNERS_ARRAY <<< "${OWNERS}"
+
+OWNERS_PARAM=$(printf ",\"%s\"" "${OWNERS_ARRAY[@]}")
```

Comment:
you need `%q` and likely want to merge this using `$IFS` or an inline Python script, please - this sort of Bash is very hard to maintain and catch edge cases